frontend/components/equipment-card.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-card border="md" class="d-flex flex-row" @click="console.log()">
+    <v-card-text class="border-e-md d-flex flex-column justify-space-between align-center">
+      <h2 class="text-h6 text-center">
+        {{ equipment.name }}
+      </h2>
+      <v-progress-circular
+        :model-value="equipment.oee"
+        size="150"
+        width="20"
+        :color="getColorState(equipment.oee)"
+        class="text-h6 mt-2">
+        <span>
+          <span :style="{fontSize: '1rem'}" class="font-weight-medium">OEE</span>
+          <br>
+          {{Math.round(equipment.oee)}}%
+        </span>
+      </v-progress-circular>
+    </v-card-text>
+
+    <v-card-text class="pa-0">
+      <v-card
+        v-for="(metric, metricIndex) in metrics"
+        :key="metric.label"
+        class="pa-0 d-flex flex-column justify-center"
+        :class="metricIndex +1 === metrics.length ? '' : 'border-b-md'"
```

Comment:
Instead of programmatically changing the class, set a standard class (like `metric-card`) and make a CSS rule to handle this:

```css
.metric-card:not(:last-child) {
    border-bottom: 2px solid var(--theme-whatever-color);
}
```

Reply:
Using the vuetify border classes ensures border properties are always consistent across the app. I don't know how but I'm sure its doable with scss, but correct solution would be to make a class for this card that applies the vuetify border class when not last-child